application/build.gradle

```diff
@@ -136,26 +139,41 @@ dependencies {
     compile group: 'org.springframework.boot', name: 'spring-boot-starter-data-rest', version: versions.springBoot
     compile group: 'org.springframework.boot', name: 'spring-boot-starter-security', version: versions.springBoot
     compile group: 'org.springframework.boot', name: 'spring-boot-starter-web', version: versions.springBoot
-    compile group: 'org.springframework.boot', name: 'spring-boot-starter-batch', version: versions.springBoot
-    compile group: 'org.springframework.hateoas', name: 'spring-hateoas', version: '0.24.0.RELEASE'
+    compile group: 'org.springframework.boot', name: 'spring-boot-starter-batch', version: versions.springBootBatch
+    compile group: 'org.springframework.hateoas', name: 'spring-hateoas', version: '0.25.1.RELEASE'
+    compile group: 'org.springframework.cloud', name: 'spring-cloud-starter-hystrix', version: '1.4.6.RELEASE'
+    compile(group: 'org.springframework.cloud', name: 'spring-cloud-starter-hystrix-dashboard', version: '1.4.6.RELEASE') {
+        exclude(module: 'logback-classic')
+    }
 
-    compile group: 'org.liquibase', name: 'liquibase-core', version: versions.liquibaseVersion
+    // Below 2 dependencies enforces older versions that do not have the vulnerabilities reported in pipeline
+    compile group: 'org.springframework.batch', name: 'spring-batch-infrastructure', version: '3.0.10.RELEASE'
```

Comment:
3.0.10.RELEASE is duplicated, could we extract to a var?

Reply:
Done. Thank you.